src/styles/components/_home-search-box.scss

```diff
@@ -1,15 +1,73 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  width: 80%;
+
```

Comment:
Add `display: flex;` so your magic `width` on the `input` and `button` will work!